data/50unattended-upgrades.Raspbian

```diff
@@ -36,6 +36,11 @@ Unattended-Upgrade::Origins-Pattern {
 //      "o=Raspbian,a=stable";
 //      "o=Raspbian,a=testing";
         "origin=Raspbian,archive=${distro_codename},label=Raspbian";
+
+        // Additionally, if you are running Raspbian on a Raspberry Pi,
+        // you probably want to uncomment the line below to match
+        // packages from the Raspberry Pi Foundation repository as well:
+//      "origin=Raspberry Pi Foundation,codename=${distro_codename},label=Raspberry Pi Foundation";
```

Comment:
Thanks for the patch!
Reading the linked documentation I believe this should be enabled by default, too.

Reply:
Yes, I wasn't sure how many Raspbian users would actually not need it, but I guess it won't hurt anyway. I can make an additional commit to uncomment that pattern then. Let me know if you want me to combine them in one new commit and rebase on master.